api/types/autoupdate/version.go

```diff
@@ -60,11 +60,13 @@ func ValidateAutoUpdateVersion(v *autoupdate.AutoUpdateVersion) error {
 		return trace.BadParameter("Spec is nil")
 	}
 
-	if v.Spec.ToolsVersion == "" {
```

Comment:
you no longer validate v.Spec.ToolsVersion. Is it used in prod? Should we keep it if not used?

Reply:
Actually we decided to rename before actual usage in prod, so this resource is not used previously